spec/requests/authenticate_user_spec.rb

```diff
@@ -44,6 +44,12 @@
       get "/works/#{work.id}"
       expect(response).to have_http_status(:success)
     end
+
+    it 'redirects from /collections/new to login URL' do
+      get '/collections/new'
```

Comment:
Should these go in the collections spec?

Reply:
I could go either way, but the current pattern concentrates all authN stuff into this request spec.

Reply:
I think we should lay these out per request (rather than per-feature).

Reply:
@jcoyne K. I'll re-organize in the morning.

Reply:
@jcoyne Done. This is ready for another look now. The request specs now line up better w/ the controllers.